requirements.txt

```diff
@@ -7,6 +7,7 @@ Flask-Bootstrap
 Flask-CORS
 path.py
 Flask-Markdown
+WTForms<3
```

Comment:
👍🏻

Reply:
Looks like before on stage! 🆗 
But, the new version does look better. So while it seems it still has some lingering update woes, it is probably good to start working towards it. But let's get a release out first.